### DISCUSSION TITLE
yealico规则移植

### DISCUSSION THREAD

**Discussion by SafariTsai** [xiaohucode/yidaRule#201] [Ideas]
有大佬会移植吗，有很多规则，或者有没有大佬知道哪里有教程我可以移植
https://wc3818290.home.blog/%E7%AB%99%E7%82%B9%E8%A7%84%E5%88%99%E5%88%97%E8%A1%A8/

**Comment by chenxingmoonset** [comment]
自己尝试写吧，会一点css和js就行。目前该站所有源都做了加密处理，只能通过网站自己制作了。

**Reply by fredsk** [reply]
那些yds例子用VScode打开全是乱码，，，我想自己写个yds结果，软件识别不了，不知道是不是有加密算法之类的，也不知到怎么解密

**Reply by chenxingmoonset** [reply]
是的，yds格式的源有加密

**Comment by gegeName** [comment]
不知道能否出一个规则实例出来而且中文注释一下，我们照着写这样子规则才会多。不然完全靠作者一个人很难有精力去维护

**Reply by QAD21** [reply]
不会css和js的话，就看其他的规则照葫芦画瓢一样可以写出来，只不过效率不高罢了

**Reply by fredsk** [reply]
那些yds例子用VScode打开全是乱码，，，我想自己写个yds结果，软件识别不了，不知道是不是有加密算法之类的

**Comment by QAD21** [comment]
关键就在于没有地方放规则的仓库，不然一人写一条都是简简单单的那种

**Reply by gegeName** [reply]
哪里有现有规则照着写啊。在哪里有呢？

**Reply by chenxingmoonset** [reply]
> 哪里有现有规则照着写啊。在哪里有呢？

内置的有精选书源的链接

**Reply by gegeName** [reply]
> > 哪里有现有规则照着写啊。在哪里有呢？
> 
> 内置的有精选书源的链接

你说的内置精选书院是App中嘛？还是哪里？

**Reply by chenxingmoonset** [reply]
> > > 哪里有现有规则照着写啊。在哪里有呢？
> 
> > 
> 
> > 内置的有精选书源的链接
> 
> 
> 
> 你说的内置精选书院是App中嘛？还是哪里？

app中管理站点=>更多=>网络导入页面，输入框下方的链接

**Reply by gegeName** [reply]
> > > > 哪里有现有规则照着写啊。在哪里有呢？
> > 
> > 
> > > 
> > 
> > 
> > > 内置的有精选书源的链接
> > 
> > 
> > 你说的内置精选书院是App中嘛？还是哪里？
> 
> app中管理站点=>更多=>网络导入页面，输入框下方的链接

这个链接打不开呀？有一次我打开了也是乱码需要什么解密？

**Reply by chenxingmoonset** [reply]
> > > > > 哪里有现有规则照着写啊。在哪里有呢？
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > 内置的有精选书源的链接
> 
> > > 
> 
> > > 
> 
> > > 你说的内置精选书院是App中嘛？还是哪里？
> 
> > 
> 
> > app中管理站点=>更多=>网络导入页面，输入框下方的链接
> 
> 
> 
> 这个链接打不开呀？有一次我打开了也是乱码需要什么解密？

点击右上角导入规则，app内的加密，会自动解析

**Reply by gegeName** [reply]
> 动解析

解析后不是变成在“管理站点”里面的资源了吗？难道是点击资源进入“编辑站点”？一个一个对着弄？

**Reply by QAD21** [reply]
> 点击右上角导入规则，app内的加密，会自动解析

大佬，有问题请教一下，目录页的请求咋写啊，
这是详细页的地址：/manga/mozhazhaohuang-shidaimanwang
这是目录页的地址：/chapterlist/mozhazhaohuang-shidaimanwang
我这样写报错：/chapterlist/$keyWord，请问咋写啊

**Reply by chenxingmoonset** [reply]
> > 动解析
> 
> 
> 
> 解析后不是变成在“管理站点”里面的资源了吗？难道是点击资源进入“编辑站点”？一个一个对着弄？

是的

**Reply by chenxingmoonset** [reply]
> > 点击右上角导入规则，app内的加密，会自动解析
> 
> 
> 
> 大佬，有问题请教一下，目录页的请求咋写啊，
> 
> 这是详细页的地址：/manga/mozhazhaohuang-shidaimanwang
> 
> 这是目录页的地址：/chapterlist/mozhazhaohuang-shidaimanwang
> 
> 我这样写报错：/chapterlist/$keyWord，请问咋写啊

使用js，将manga替换为chapterlist即可

**Reply by QAD21** [reply]
> 使用js，将manga替换为chapterlist即可
解决了，但是又有新问题了，就是源代码里获取不了目录和正文，有没有办法从控制台获取啊？
![q12](https://github.com/user-attachments/assets/e91183c5-6923-4b33-a3dd-614b3d8287df)

**Reply by chenxingmoonset** [reply]
> > 使用js，将manga替换为chapterlist即可
> 
> 解决了，但是又有新问题了，就是源代码里获取不了目录和正文，有没有办法从控制台获取啊？
> 
> ![q12](https://github.com/user-attachments/assets/e91183c5-6923-4b33-a3dd-614b3d8287df)
> 
> 
> 
> 

有可能是动态加载的，可以抓包看看是否有目录或正文api，或者使用浏览器打开获取cookie再请求

**Reply by QAD21** [reply]
> 有可能是动态加载的，可以抓包看看是否有目录或正文api，或者使用浏览器打开获取cookie再请求
这是动态加载吧？目录和正文的API都能找到，要不大佬写个，我来照着学学？，包子漫画：https://bun.godamh.com
![QQ_1721963090123](https://github.com/user-attachments/assets/09b532f1-e989-4ad1-87f8-0ebb34b0f526)
